homeassistant/components/upnp/__init__.py

```diff
@@ -49,9 +45,7 @@
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.Schema(
-            {
-                vol.Optional(CONF_LOCAL_IP): vol.All(ip_address, cv.string),
-            },
+            {},
```

Comment:
Why are we changing the config schema? Is configuration.yaml not deprecated for this integration? Why should we have an empty dict as validator? Is this a breaking change?